### PR TITLE
Add backstage to kndp platform, configure backstage to work with kndp

### PR DESCRIPTION
**Pull request by catalindev0ps**

### CODE REVIEW

**Comment by web7evgheni** [comment]
@catalindev0ps Please move backstage outside of kndp package, to be placed right in ./packages

**Comment by catalindev0ps** [comment]
@web7evgheni backstage is already at root/packages/backstage

**Review by web7evgheni — CHANGES_REQUESTED** [review]
@catalindev0ps Please double check, because i see it in ./packages/kndp

**Comment by catalindev0ps** [comment]
@web7evgheni now it's fixed

File: yarn.lock
Comment (web7evgheni) [inline yarn.lock:1]: @catalindev0ps Please remove and add to ignore yarn.lock

File: package.json
Comment (web7evgheni) [inline package.json:1]: @catalindev0ps Please remove and add to ignore package-lock.json

**Review by web7evgheni — CHANGES_REQUESTED** [review]

**Review by web7evgheni — APPROVED** [review]

**Pull request merged**